### DISCUSSION TITLE
Persistency FT [General Information]

### DISCUSSION THREAD

**Discussion by antonkri** [eclipse-score/score#116] [Persistency FT]
FT facts:
- Lead: @umaucher 
- Proxy: `open`
- Slack channel: [#score-persistency](https://sdvworkinggroup.slack.com/archives/C08B339ETQU)
- Repo: https://github.com/eclipse-score/persistency
- Issue handling: https://github.com/eclipse-score/persistency/issues
- FT meeting minutes: https://github.com/orgs/eclipse-score/discussions/2518?sort=new
- GitHub project: https://github.com/orgs/eclipse-score/projects/20/views/4 (?? tbc)

Contribution/Feature Request:
- [KVS](https://github.com/eclipse-score/score/pull/219)
- [File Access](https://github.com/eclipse-score/score/pull/220)

- Following links can be also helpful to you:
  - [SCORE main page](https://eclipse-score.github.io/)
  - [Get Involved](https://eclipse-score.github.io/get_involved.html#get-involved) 
  - [Technical Documentation](https://eclipse-score.github.io/score/)
  - [Project Handbook](https://eclipse-score.github.io/score/platform_management_plan/project_management.html)
  - [SCORE releases & roadmap](https://eclipse-score.github.io/score/score_releases/index.html)

**Comment by guysagnes** [comment]
### Updated information

- Contributors
  - @SorinMold
  - @guysagnes - until 31.07.2025 
  - @sbachmann-qorix 
  - @joshualicht 
  - @vinodreddy-g 
  - @umaucher 
  - @ahmed-elsaka-JC
  - @vbogdanb
  - @Jannikszwajczyk
  
Weekly Thursday 14:30 - 16:00 CE(S)T
[Join the meeting now](https://teams.microsoft.com/l/meetup-join/19%3ameeting_NDZjZTgwMjgtNjk3ZC00YTBkLTkwZjEtZjdiYjJlYTM5Mjlm%40thread.v2/0?context=%7b%22Tid%22%3a%228d4b558f-7b2e-40ba-ad1f-e04d79e6265a%22%2c%22Oid%22%3a%222e01a274-5b7e-48f0-bd9a-2e637bbfd72f%22%7d) 
Meeting ID: 342 596 516 241 6 
Passcode: ak2ky95Z
 
 - Slack channel: https://sdvworkinggroup.slack.com/archives/C08B339ETQU

 - Repo: https://github.com/eclipse-score/inc_mw_per

- Contribution/Feature Request:

  - https://github.com/eclipse-score/score/issues/95
    - https://github.com/eclipse-score/score/pull/1067 
    - https://github.com/eclipse-score/score/pull/1027
    - https://github.com/eclipse-score/score/pull/219
    - https://github.com/eclipse-score/score/pull/220
      - File system: roadmap Release 1.0 / to be aligned with AoU (Assumption of Use) Operating System

### Following links can be also helpful to you:

  - [SCORE main page](https://eclipse-score.github.io/)
  - [Get Involved](https://eclipse-score.github.io/get_involved.html#get-involved)
  - [Technical Documentation](https://eclipse-score.github.io/score/)
  - [Project Handbook](https://eclipse-score.github.io/score/main/platform_management_plan/index.html)
  - [SCORE releases & roadmap](https://eclipse-score.github.io/score/main/score_releases/index.html)
  - [SCORE Process Areas](https://eclipse-score.github.io/process_description/main/process_areas/index.html)
    - [SCORE Building blocks concept](https://eclipse-score.github.io/process_description/main/general_concepts/score_building_blocks_concept.html)
    - [SCORE Traceability concept](https://eclipse-score.github.io/process_description/main/general_concepts/score_traceability_concept.html)
